jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/testutils/payload/PayloadCreator.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.homeoffice.digital.sas.jparest.testutils.payload;
+
+import com.google.gson.Gson;
+
+import java.io.Serializable;
+import java.util.Map;
+
+public final class PayloadCreator {
+
+    private PayloadCreator(){
+        // no instantiation
+    }
+
+    public static <S extends Serializable> String createPayload(Map<String, S> fieldHashMap){
+        return new Gson().toJson(fieldHashMap);
+    }
```

Comment:
Just a small suggestion if you have the time, I noticed theres a few calling methods that only have one field,value.
We could overload this method to take a single field, value so that those particular callers don't even need to wrap with Map.of() 

```suggestion
    }
    
    public static <S extends Serializable> String createPayload(String field, S value){
        return createPayload(Map.of(field, value));
    }
```

Reply:
I thought about adding an overloaded single field/value method as well, so I think it's worth adding, added.

Reply:
Nice one :ok_hand: